state/azure/cosmosdb/cosmosdb.go

```diff
@@ -335,6 +336,10 @@ func (c *StateStore) Multi(request *state.TransactionalStateRequest) error {
 	return nil
 }
 
+func (c *StateStore) Ping() error {
+	return errors.New("not implemented")
```

Comment:
Probably, state stores that are in GA should have this feature implemented and added to conformance tests.

Reply:
I mean, to also add "ping()" validation to conformance tests.

Reply:
@artursouza 
I try to add a conformance test. PTAL.